query/src/test/java/org/infinispan/query/blackbox/ClusteredCacheTest.java

```diff
@@ -83,7 +84,7 @@ public ClusteredCacheTest() {
       return new Object[]{
             new ClusteredCacheTest().storageType(StorageType.OFF_HEAP),
             new ClusteredCacheTest().storageType(StorageType.BINARY),
-            new ClusteredCacheTest().storageType(StorageType.OBJECT),
+            new ClusteredCacheTest().storageType(StorageType.OBJECT)
```

Comment:
fyi, those commas are left at the end of list to prevent modifying the line when you add another test